src/views/Details.vue

```diff
@@ -0,0 +1,247 @@
+<template>
```

Comment:
What is this?

Reply:
This is a file I was using during my tests that I forgot to delete 😅